Library/Homebrew/style.rb

```diff
@@ -178,6 +180,12 @@ def run_shellcheck(files, output_type)
         *files,
       ]
 
+      if fix
+        patches = system_command(shellcheck, args: ["--format=diff", *args]).stdout
+        patch_command = %w[patch -g 0 -f -d / -p0]
```

Comment:
Can you:
- use long flags where possible
- add a comment to explain why this is needed and what these flags are doing

Thanks!